ci_scripts/sync_board_database.py

```diff
@@ -172,6 +165,13 @@ def main(args: argparse.Namespace) -> int:
         else:
             news_file_text = "Offline board database created."
 
+        pr_info = PullRequestInfo(
+            repo="ARMMbed/mbed-tools",
+            head_branch=args.head_branch,
+            base_branch=args.base_branch,
+            subject=args.pr_subject,
+            body=news_file_text,
+        )
```

Comment:
Could you commend why using news file text as the PR description is useful? What motivates the change?

Reply:
Will do.